### PR TITLE
Make ::selection allowed-properties list match spec

### PR DESCRIPTION
**Pull request by hovdallakyan**
Replaced the list of the accepted properties to use with ::selection as described in spec 
https://drafts.csswg.org/css-pseudo-4/#highlight-styling

### CODE REVIEW

File: files/en-us/web/css/_doublecolon_selection/index.html [hunk 1]
```diff
@@ -23,7 +23,6 @@ <h2 id="Allowable_properties">Allowable properties</h2>
 <ul>
```
Comment (sideshowbarker) [inline files/en-us/web/css/_doublecolon_selection/index.html:23]: I notice that at https://drafts.csswg.org/css-pseudo-4/#highlight-styling the spec lists:

* color
* background-color
* text-decoration and its associated properties
* text-shadow
* stroke-color, fill-color, and stroke-width

So, along with `cursor`, as far as properties in this MDN article that aren’t in the spec, the spec lacks the following:

* caret-color
* outline
* text-emphasis-color

So, shouldn’t we remove those from the MDN article?

And as far properties that aren’t in this the MDN article, but are in the spec, the spec has the following:

* stroke-color, fill-color, and stroke-width

So, shouldn’t we add those?

In other words, shouldn’t we just replace the list that we currently have in the MDN with the following:

* color
* background-color
* text-decoration and its associated properties
* text-shadow
* stroke-color, fill-color, and stroke-width

Comment (HDallakyan) [reply files/en-us/web/css/_doublecolon_selection/index.html:23]: totally agree

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Hovhannes, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉
